src/main/java/com/youtube/MyParseYoutube.java

```diff
@@ -0,0 +1,66 @@
+package com.youtube;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentSnippet;
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.api.services.youtube.model.VideoListResponse;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.youtube.YoutubeService.getService;
+
+public class MyParseYoutube {
+    private static final String DEVELOPER_KEY = "AIzaSyDBoAaMpOt0hDvr7wp_Iu2UFL0ZPb3c8oQ";
```

Comment:
It is not good practice to hard code your credentials. It is better to put them into the properties file.